client/src/components/LoginSession/LoginSession.js

```diff
@@ -146,4 +131,22 @@ LoginSession.propTypes = {
     LogOutEndpoint: PropTypes.string.isRequired,
 };
 
-export default LoginSession;
+LoginSession.defaultProps = {
+    displayToastSuccess: () => 0,
+    displayToastFailure: () => 0,
```

Comment:
That's kind of a weird default here. I would just do an existence check before calling the method.

Also, you should still define those entries on `propTypes` even when you provide defaults for them.

Reply:
Updated